cmd/if-mover/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+const (
+	ifBase     = "mlink"
+	hostNsFile = "/tmp/hostns.txt"
+	multusLink = "net1"
+)
+
+func main() {
+
+	log.Println("Determining host namespace id")
+	// The Host network namespace name comes from a file generated by the following command:
+	// ip --json netns list-id | jq -j ".[] | select(.nsid == 0).name" > /tmp/hostns.txt
```

Comment:
Despite my other comment below, why not call `ip` (hopefullysomething else) from this go program instead. We could get better error handling if the host ns cannot be discovered.